practices/practice-two/src/styles/sections/header.css

```diff
@@ -0,0 +1,251 @@
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.banner {
+  background: linear-gradient(
+      360deg,
+      rgba(0, 0, 0, 5e-5) 0%,
+      rgba(0, 0, 0, 0.5) 100%
+    ),
+    url(../../assets/images/header-bg.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.header-wrapper {
+  font-family: var(--font-mont);
+  padding: 15px 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 1px 0 var(--seventh-color);
+}
+
+.header-item {
+  display: flex;
+  align-items: center;
+}
+
+.header-item-logo {
+  font-weight: 400;
+  font-size: 28.75px;
+  line-height: 39px;
+  text-transform: uppercase;
+  padding-right: 10px;
+}
+
+.header-item-phone {
+  display: inline-block;
+  padding-left: 20px;
+  position: relative;
+}
+
+.header-item-phone::after {
+  position: absolute;
+  content: "";
+  width: 1px;
+  height: 20px;
+  background-color: var(--fourth-color);
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.header-item-phone-icon {
+  padding-right: 5px;
+}
+
+.c-md-navigation,
+.c-xl-navigations {
+  display: none;
+}
+
+.c-xl-navigations-item-link,
+.c-md-navigations-item-link {
+  text-transform: uppercase;
+  color: var(--seventh-color);
+}
+.c-xl-navigations-item-btn {
+  padding: 10px 15px;
+}
+
+.header-introduce-wrapper {
+  display: flex;
+  align-items: center;
+  font-family: var(--font-robo);
+  color: var(--ninth-color);
+}
+
+.header-introduce-item {
+  flex: 1;
+}
+
+.header-introduce-item-title {
+  font-weight: 400;
+  font-size: 38px;
+  line-height: 45px;
+}
+
+.header-introduce-item-description {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  padding: 10px 0 15px;
+}
+
+.header-introduce-item-btn {
+  padding: 20px 50px;
+}
+
+/*
+mobile devices - sm
+ */
+@media screen and (min-width: 320px) and (max-width: 480px) {
+  .header-item-phone {
+    display: none;
+  }
+
+  .header-introduce-wrapper {
+    height: 300px;
+  }
+
+  .header-introduce-item:last-child {
+    display: none;
+  }
+
+  .header-introduce-item-title {
+    font-weight: 400;
+    font-size: 28px;
+    line-height: 53px;
+  }
+
+  .header-introduce-item-description {
+    font-weight: 300;
+    font-size: 13px;
+    line-height: 24px;
+  }
+
+  .header-introduce-item-btn {
+    padding: 15px 30px;
+  }
+}
+
+/*
+iPads & tablets - md
+*/
+@media screen and (min-width: 481px) and (max-width: 768px) {
+  .header-introduce-wrapper {
+    height: 500px;
+  }
+  .header-introduce-item:first-child {
+    flex: 2;
+  }
+
+  .header-introduce-item:last-child {
+    flex: 1;
+  }
+}
+
+/*
+styles common mobile and ipads: c-md
+*/
+@media screen and (min-width: 320px) and (max-width: 768px) {
+  .c-md-navigation {
+    position: relative;
+    display: block;
+  }
+
+  .c-md-navigation-icon {
+    font-size: 25px;
+    color: var(--ninth-color);
+  }
+
+  #checkbox:checked ~ .c-md-navigations {
```

Comment:
Why we have styles for id selector?

Reply:
I use it to hide and show the navigation bar on mobile

Reply:
can we use class?

Reply:
I think not possible, because I am using label tag and input tag type checkbox instead of js